src/qilib/utils/storage/mongo.py

```diff
@@ -307,6 +304,21 @@ def tag_in_storage(self, tag: TagType) -> bool:
                 parent = doc['_id']
         return True
 
+    @staticmethod
+    def _encode_field(field: Union[int, str]) -> str:
+        """Encodes a field value
```

Comment:
Doesn't add anything to the method name. Maybe tell what a field is, so somebody can understand if he/she should use this method or not (it references to an individual field in a structure that is updated or read or something like that).

Reply:
Updated the doc string